Updated downstream version numbers from 4.5 to 4.6.

adahms opened the pull request.

dayleparker commented on doc-Appliance_Hardening_Guide/miq/build/en-US/master.xml:
```diff
@@ -2,7 +2,7 @@
 <!ENTITY % sgml.features "IGNORE">
 <!ENTITY % xml.features "INCLUDE">
-<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/dbcentx.mod">
+<!ENTITY % DOCBOOK_ENTS PUBLIC "-//OASIS//ENTITIES DocBook Character Entities V4.6//EN" "http://www.oasis-open.org/docbook/xml/4.6/dbcentx.mod">
```
Hi @adahms - looks like the find and replace picked up the Docbook version as well -- should this remain as 4.5?

dayleparker commented on doc-Managing_Providers/topics/Viewing_Network_Providers_Timeline.adoc:
```diff
@@ -1,6 +1,6 @@
 = Viewing a Network Provider's Timeline
 
-//SME feedback - leave section out of 4.5 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
+//SME feedback - leave section out of 4.6 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
```
This perhaps should be changed back to 4.5, but am unsure about this note.

dayleparker commented on doc-Managing_Providers/topics/Viewing_Swift_Manager_Timeline.adoc:
```diff
@@ -1,7 +1,7 @@
 = Viewing a Swift Manager’s Timeline
 
-//SME feedback - leave section out of 4.5 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
+//SME feedback - leave section out of 4.6 release. See https://bugzilla.redhat.com/show_bug.cgi?id=1382585#c7
```
This perhaps should be changed back to 4.5, but am unsure about this note.

dayleparker commented on doc-Provisioning_Virtual_Machines_and_Hosts/topics/Provisioning_Requests.adoc:
```diff
@@ -480,7 +480,7 @@ include::common/provisioning-requests-openstack.adoc[]
 ==== Provisioning a Google Compute Engine Instance from an Image
 
-From Red Hat {product-title} 4.5, a Google Compute Engine instance can be provisioned. Some functionality available in other cloud providers is currently unavailable when provisioning from Google Compute Engine.
+From Red Hat {product-title} 4.6, a Google Compute Engine instance can be provisioned. Some functionality available in other cloud providers is currently unavailable when provisioning from Google Compute Engine.
```
This should be reverted to 4.5 (or possibly 4.2), I believe.

dayleparker commented on methods_available_for_automation/Methods_Available_for_Automation.adoc:
```diff
@@ -215,17 +215,17 @@ The result of dumpRoot is below. The value of miq_server is what gets
-[----] I, [2012-10-23T13:53:54.536238 #5320:ef8c538]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> Attributes - vmdb_object_type: vm
-[----] I, [2012-10-23T13:53:54.537159 #5320:f329024]  INFO -- : <User-Defined Method> [InspectMe] - Root:<$evm.root> End Attributes
-[----] I, [2012-10-23T13:53:54.537772 #5320:f329024]  INFO -- : <User-Defined Method>
+[----] I, [2012-10-23T13:53:54.617279 #5320:f329024]  INFO -- : <User-Defined Method> [InspectMe] - EVM Automate Method Started
```
@adahms Should these sixes be changed back to fives?

dayleparker commented:
Hi @adahms,
There are a few things picked up by the find & replace which might need a closer look at/editing. I made an edit to a note I put in myself in the HA guide, but I think it's better if you review these other items before we merge. Everything else looks good though!
Cheers,
Dayle

dayleparker commented:
Hi @adahms,
There are a few things picked up by the find & replace which might need a closer look at/editing. I made an edit to a note I put in myself in the HA guide, but I think it's better if you review these other items before we merge. Everything else looks good though!
Cheers,
Dayle

adahms commented:
@dayleparker - Thanks for the review!

I have now addressed your comments by reverting some references and changing others to '4.2' instead of '4.6'.

Let me know if you spot anything else!

dayleparker commented:
Thanks for making those last edits, Andrew! I gave the PR one final review and everything looks great :)
Cheers,
Dayle
